src/app/client/src/app/modules/groups/services/groups/groups.service.ts

```diff
@@ -85,5 +85,12 @@ export class GroupsService {
     return await this.groupCservice.getAll().toPromise();
   }
 
+  async getGroupDetails(groupid) {
```

Comment:
why can't we use observables here

Reply:
This is the example given in client-service so same way I have used

Reply:
Done